tests/test_examples.py

```diff
@@ -52,6 +52,15 @@ def test_SOLOVEV_vacuum(SOLOVEV_vac):
     np.testing.assert_allclose(data["iota"], 0, atol=1e-16)
     np.testing.assert_allclose(data["|J|"], 0, atol=3e-3)
 
+    # test that solving with the continuation method works correctly
+    # when eq resolution is lower than the mres_step
+    eq.change_resolution(L=3, M=3)
```

Comment:
I feel like in this sort of case it would be faster/better to just solve at the full resolution from the start, instead of doing multigrid in M

Like if we did
```python
Mi = min(M, mres_step) if mres_step > 0 else M
mres_steps = int(max(np.ceil(M / mres_step), 1)) if mres_step > 0 else 0